integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/TransferDashFragment.kt

```diff
@@ -172,6 +172,19 @@ class TransferDashFragment : Fragment(R.layout.transfer_dash_fragment) {
             )
         }
 
+        enterAmountToTransferViewModel.dashWalletNotSyncedCallback.observe(viewLifecycleOwner){
+            val dashAccountEmptyError = CoinbaseGenericErrorUIModel(
```

Comment:
It might be better to use an `AdaptiveDialog` here. Some advantages:
- Auto dismissal on the lock screen.
- Clear flow: immediately obvious what happens when the user chooses positive/negative action.
- Easier to add additional code for the positive action (like analytics).
- Universal design that can be easily changed for all dialogs.

Reply:
Agreed.  Though new designs have to dialog when syncing.